rcldotnet_common/DllLoadUtils.cs

```diff
@@ -231,10 +231,10 @@ public class DllLoadUtilsWindowsDesktop : DllLoadUtilsAbs
             [DllImport("kernel32.dll", EntryPoint = "LoadLibraryA", SetLastError = true, ExactSpelling = true)]
             private static extern IntPtr LoadLibraryA(string fileName, int reserved = 0);
 
-            [DllImport("kernel32.dll", SetLastError = true, ExactSpelling = true)]
+            [DllImport("kernel32.dll", EntryPoint = "FreeLibrary", SetLastError = true, ExactSpelling = true)]
```

Comment:
Seems this was broken by this commit: https://github.com/ros2-dotnet/ros2_dotnet/commit/3bd4a55dd795c413e563c88cae9381ef1dc41f5d

Could you add the `EntryPoint` for UWP here as well?

Reply:
fixed this :)